core/build.gradle

```diff
@@ -257,3 +265,38 @@ startScripts {
         }
     }
 }
+
+def getVersionName = { ->
+    if (System.getenv("DOCKER_VERSION")) {
```

Comment:
This is when you locally want to create a Docker image with a custom version? It would be good if this was documented somewhere.